src/datatypes/dtype.rs

```diff
@@ -178,6 +182,14 @@ impl DataType {
                 Box::new(Field::new(field.name.clone(), field.dtype.to_physical())),
                 *size,
             ),
+            Image(field) => List(Box::new(Field::new(
```

Comment:
This internal representation will actually need to be a `pa.struct({"data": pa.list_(data_dtype), "shapes": pa.list_(pa.uint64())})` in order to be able to roundtrip with the Python representation, still adding that.